### PR TITLE
Add PerSenT

### PR DESCRIPTION
**Pull request by jeromeku**
Added [Person's SentimenT](https://stonybrooknlp.github.io/PerSenT/) dataset.

### CODE REVIEW

File: datasets/per_sent/README.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
+---
+YAML tags:
```
Comment (lhoestq) [inline datasets/per_sent/README.md:2]:
```suggestion
```

File: datasets/per_sent/README.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+- 1K<n<10K
+source_datasets:
+- extended|other-MPQA-KBP Challenge-MediaRank
+task_categories:
+- text-classification
+task_ids:
+- sentiment-classification
+---
+
+# Dataset Card for [Dataset Name]
```
Comment (lhoestq) [inline datasets/per_sent/README.md:23]:
```suggestion
# Dataset Card for PerSenT
```

**Review by lhoestq — COMMENTED** [review]
Good job ! thank for for adding this one
And good dataset card as well

One of the two dummy_data.zip file is unnecessary. 
Could you remove datasets/per_sent/dummy_data.zip ?

File: datasets/per_sent/per_sent.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+                    "filepath": dev_path,
+                    "split": "dev",
+                },
+            ),
+        ]
+
+    def _generate_examples(self, filepath, split):
+        """ Yields examples. """
+
+        with open(filepath) as f:
```
Comment (lhoestq) [inline datasets/per_sent/per_sent.py:143]: could you add the encoding here ? for example `encoding="utf-8"`

Comment (jeromeku) [reply datasets/per_sent/per_sent.py]: Removed the extraneous zip file and fixed encoding.  CI failing on [run_dataset_script_tests_pyarrow_1_WIN](https://circleci.com/gh/huggingface/datasets/7347?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)?

**Comment by lhoestq** [comment]
looks like this PR contains changes in many other files than the ones for PerSenT
can you create another branch and another PR ?

**Comment by lhoestq** [comment]
closing since #1142 was merged

**Pull request closed without merging**